openqaoa/devices.py

```diff
@@ -299,23 +301,25 @@ def __init__(self, device_name: str, s3_bucket_name: str = None,
 
         Parameters
         ----------
-		device_name: `str`
-			The ARN string of the braket QPU/simulator to be used
-        s3_bucket_name: `str`
+          device_name: `str`
+            The ARN string of the braket QPU/simulator to be used
+          s3_bucket_name: `str`
             The name of S3 Bucket where the Braket run results will be saved.
-        aws_region: `str`
+          aws_region: `str`
             The aws region in which the QPU/simulator is located. Defaults to the
             region set in aws cli config.
-        folder_name: `str`
-            The name of the folder in the s3 bucket that will contain the results
-            from the tasks performed in this run.
+          folder_name: `str`
+              The name of the folder in the s3 bucket that will contain the results
+              from the tasks performed in this run.
         """
         
         self.device_name = device_name
         self.device_location = 'aws'
         self.s3_bucket_name = s3_bucket_name
         self.aws_region = aws_region
         self.folder_name = folder_name
+        self.s3_bucket_name = s3_bucket_name
```

Comment:
repetition for `self.s3_bucked_name`, `self.aws_region`

Reply:
Damn! I'll fix them